docs/source/index.rst

```diff
@@ -1,6 +1,6 @@
 NuCypher
 ========
-*A proxy re-encryption network to empower privacy in decentralized systems*
+*A decentralized cryptological network offering accessible, intuitive, and collusion-resistant runtimes and interfaces to novel and exciting cryptographic primitives.*
```

Comment:
I dig > a  decentralized cryptological network.  However, this reads a bit like a run-on sentence to me.  Is it possible to reduce `runtimes and interfaces` into a more dense noun?

Reply:
how about services?

Reply:
I think it's important to maintain some form of `<a>` and `<b>` construction, where one describes the actual, running processes available to a dapp developer audience ("runtimes") and the other describes the codebase which is available for dispassionate inspection ("interfaces").  

I'm not at all married to these particular words, but I think it's important to indicate in some fashion that we endeavour to provide *both* of these, rather than one or the other.

Reply:
I'm not sure I agree that the latter is as important as the former.

> the other describes the codebase which is available for dispassionate inspection ("interfaces").
Why is this as important as the "the actual, running processes available to a dapp developer audience"? Yes, we strive to provide a clear, readable codebase but it's in pursuit of providing a useful network not the end goal in and of itself.

A network with a beautiful codebase but no adoption would be disappointing and unsustainable, whereas I don't think the reverse is true.